pom.xml

```diff
@@ -411,8 +424,14 @@
         </plugin>
       </plugins>
     </pluginManagement>
-
   </build>
+  <reporting>
```

Comment:
Experimenting with plugin documentation generation. To be completed in JAVA-3301